benchmarks/ctia/entity/actor_bench.clj

```diff
@@ -0,0 +1,31 @@
+(ns ctia.entity.actor-bench
+  (:require [ctia.test-helpers
+             [benchmark :refer [cleanup-ctia!
+                                setup-ctia-es-store!]]
+             [core :as helpers :refer [delete post]]]
+            [ctim.examples.actors
+             :refer [new-actor-minimal
+                     new-actor-maximal]]
+            [perforate.core :refer :all]))
+
+(def small-actor new-actor-minimal)
+(def big-actor (dissoc new-actor-maximal :id))
+
+(defgoal create-actor "Create Actor"
+  :setup (fn [] [(setup-ctia-es-store!)])
+  :cleanup (fn [_] cleanup-ctia!))
+
+(defn play [fixture]
+  (let [{:keys [status parsed_body]}
+        (post "ctia/actor"
+              :body fixture
+              :headers {"Authorization" "45c1f5e3f05d0"})]
+    (if (= 201 status)
+      (delete (str "ctia/actor" (:id parsed_body)))
+      (prn "play-big: " status))))
+
+(defcase create-actor :big-actor-es-store
+  [_] (play big-actor))
```

Comment:
`play` does not only create but also either delete or print to stdout so I think it doesn't really measure only the time it takes to call `create-actor`. Why is the `delete` necessary, isn't it handled by the `cleanup-ctia!`?

I would either rename `create-actor` as `create-and-delete-actor` or remove completely the `if` body to remove the `delete` and `prn` statements.